### DISCUSSION TITLE
Proxmox LXC Updater - add option Skip stopped containers

### DISCUSSION THREAD

**Discussion by ramess18** [tteck/Proxmox#1157] [Ideas]
Hi,
I'd like to thank you for your fantastic Proxmox scripts set and I have one request for you. I love Proxmox LXC Updater and use it regulary. Latest releases missing the option Skip stopped containers. Could you please return this option or make other version (ie. Proxmox LXC Updater - Skip stopped)?
Thank you
Radim

**Comment by tteck** [comment]
The script's current version will skip template containers by default, which is why the skip stopped containers option was added originally. If you need to skip containers other than templates, you can add their container IDs to the end of the command. For instance, you can use this command: 
```
bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/misc/update-lxcs.sh)" -s 100 102 118
```
  to skip all templates plus containers 100, 102, and 118.

While it's recommended to use the latest version of the LXC Updater, if you still want to use the older version, you can go to this [link](https://raw.githubusercontent.com/tteck/Proxmox/44c614f65dc849b4331920926b1f15e55b1de2bb/misc/update-lxcs.sh) to copy the code and save it locally. Once you have the code, you can execute it whenever you like.

**Reply by ramess18** [reply]
Thanks for the command tip with arguments. I've been using older version as you wrote.

I can see it as example in the script's description now.
